packages/tradershub/src/components/Modal/ModalContent.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactNode } from 'react';
+import { qtMerge } from '@deriv/quill-design';
+
+/**
+ * Type for the ModalContent component props
+ * @typedef TModalContent
+ * @property {ReactNode} children - Children nodes
+ * @property {string} [className] - Optional CSS class name
+ */
+type TModalContent = {
+    children: ReactNode;
+    className?: string;
+};
```

Comment:
Create a generic type for modal and use it here. Same goes to Header and Footer. Can pick or omit if needed